Document variable behavior for local exec mode (#709)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/variables/managing-variables.mdx

The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace. This is where you can add, edit, and delete workspace-specific variables. You can also apply and remove variable sets from the workspace.

> The **Variables** page is not available for workspaces configured with `Local` [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode). Workspace variables defined in HCP Terraform are not evaluated in local execution mode.

Reviewer: ```suggestion
The **Variables** page is not available for workspaces configured with `Local` [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode). HCP Terraform does not evaluate workspace variables or variable sets in local execution mode.
```

Only members of the [Owners Team](/terraform/cloud-docs/users-teams-organizations/permissions#organization-owners) or members of a team with the [Manage all projects](/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-projects) or [Manage all workspaces](/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-workspaces) permission can create, update, and delete variable sets.

> Variable sets are not evaluated during Teraform runs for workspaces configured with `Local` [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode).

Reviewer: ```suggestion
HCP Terraform does not evaluate variable sets during Terraform runs for workspaces configured with `Local` [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode).
```
